sql/telemetry_derived/main_summary_v4/metadata.yaml

```diff
@@ -9,7 +9,8 @@ labels:
 scheduling:
   dag_name: bqetl_main_summary
   start_date: '2019-10-25'
+  priority: 90
```

Comment:
Manually increasing the priority for main_summary so that it gets scheduled first

Reply:
We might want to also increase priority for clients_daily and clients_last_seen (and maybe the core and baseline variants), since many other DAGs have sensors that wait for those.